internal/rollup/package.json

```diff
@@ -5,7 +5,9 @@
         "rollup": "0.60.1",
         "rollup-plugin-node-resolve": "3.3.0",
         "rollup-plugin-sourcemaps": "^0.4.2",
+        "source-map": "0.5.7",
```

Comment:
again, is this really require'd somewhere in our code?

Reply:
Removed these in favor of using @build_bazel_rules_nodejs_node_loader_deps//:node_modules for the source-map-support require in node_loader.js